app/controllers/UnloadingFindingsController.scala

```diff
@@ -43,4 +44,16 @@ class UnloadingFindingsController @Inject() (
         val viewModel = viewModelProvider.apply(request.userAnswers)
         Ok(view(request.userAnswers.mrn, arrivalId, viewModel))
     }
+
+  def onSubmit(arrivalId: ArrivalId): Action[AnyContent] =
+    actions.getStatus(arrivalId) {
+      implicit request =>
+        if (request.userAnswers.get(DoYouHaveAnythingElseToReportYesNoPage).isDefined) {
+          Redirect(controllers.routes.CheckYourAnswersController.onPageLoad(arrivalId))
+        } else {
+          Redirect(
+            controllers.routes.AddCommentsYesNoController.onPageLoad(arrivalId, NormalMode)
+          )
+        }
```

Comment:
I think just redirect them to `AddCommentsYesNoController` every time - I can live with that. If they answered that question and clicked back to the cross-check page without answering any others it would then take them to the CYA and those questions would never get asked